### ISSUE TITLE
Route doesn't match on file://

### ISSUE THREAD

**Issue by rahatarmanahmed** [yoshuawuyts/sheet-router#66]
Ran into this issue using `choo@4.0.1-0` in Electron. Works fine in `choo@3`. (I think this is a sheet-router issue)

```
Uncaught Error: route '/Users/rahat/Downloads/test/test.html' did not match
```

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>
    <script>
      const choo = require('choo')
      const html = require('choo/html')

      const app = choo()
      app.model({
        state: {
          value: 'wow'
        }
      })

      const view = state => html`<div>${state.value}</div>`

      // choo@4.0.1-0
      app.router('/', [
        ['/', view]
      ])

      // choo@3.3.0
      // app.router(route => route('/', view))

      const tree = app.start()
      document.body.appendChild(tree)
    </script>
  </body>
</html>

```

To reproduce:
1. `npm install -g electron`
2. `electron test.html` (the test html above)

You could also bundle the above JS and load it in Chrome with `file://`. Is there a way to make choo/sheet-router assume only hashes for situations like this?

**Comment by rahatarmanahmed** [comment]
On closer inspection, this seems like a choo issue. I'll close this and open an issue there.

**Comment by rahatarmanahmed** [comment]
No, I spoke too soon. There's already [code to handle electron](https://github.com/yoshuawuyts/sheet-router/blob/master/_pathname.js#L2) but it only works when the file is named `index.html`. If it's not feasible to allow other filenames, maybe this should be documented somewhere?

**Comment by yoshuawuyts** [comment]
yeah sure we could allow more filenames

**Comment by yoshuawuyts** [comment]
e.g. PR welcome!

**Comment by ungoldman** [comment]
Routing fails in electron when trying to navigate anywhere past initial `index.html` as well. @rahatarmanahmed's fix (#67) works for me after testing locally.
